site/src/sphinx/setup-configuration.rst

```diff
@@ -110,14 +111,21 @@ Core properties
   - the number of worker threads dedicated to handling repository reads and writes.
     If ``null``, the default value of '16 threads' is used.
 
-- ``cacheSpec`` (string)
+- ``repositoryCacheSpec`` (string)
 
   - the cache specification string which determines the capacity and behavior of the repository
     access cache. Refer to `the Caffeine API documentation
     <https://static.javadoc.io/com.github.ben-manes.caffeine/caffeine/2.5.5/index.html?com/github/benmanes/caffeine/cache/CaffeineSpec.html>`_
     for more information. Note that the weight of the cache has been tuned to be roughly proportional to its
     memory usage.
 
+- ``sessionCacheSpec`` (string)
+
+  - the cache specification string which determines the capacity and behavior of the cache for the session
+    information of the server. Refer to `the Caffeine API documentation
+    <https://static.javadoc.io/com.github.ben-manes.caffeine/caffeine/2.5.5/index.html?com/github/benmanes/caffeine/cache/CaffeineSpec.html>`_
```

Comment:
Instead of repeating the same URL twice, you could extract it into the beginning of the `.rst` file.